ansible/roles/kong-api/defaults/main.yml

```diff
@@ -4763,4 +4763,21 @@ kong_apis:
       config.hour: "{{ medium_rate_limit_per_hour }}"
       config.limit_by: credential
     - name: request-size-limiting
-      config.allowed_payload_size: "{{ small_request_size_limit }}"
\ No newline at end of file
+      config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: listForm
+      uris: "{{ data_service_prefix }}/v1/form/list"
+      upstream_url: "{{ player_service_url }}/plugin/v1/form/list"
+      strip_uri: true
+      plugins:
+      - name: cors
+      - "{{ statsd_pulgin }}"
+      - name: acl
+        config.whitelist:
+          - 'groupAdmin'
+      - name: rate-limiting
+        config.policy: local
+        config.hour: "{{ medium_rate_limit_per_hour }}"
```

Comment:
give small rate limit per hour, as we will not be calling this API frequently

Reply:
Done